packages/core/test/unit/plugins/default/dataTable.test.ts

```diff
@@ -0,0 +1,63 @@
+import * as cheerio from 'cheerio';
+import dataTable from '../../../../src/plugins/dataTable';
+
+test('postRender should add appropriate classes and attributes to m-table elements', () => {
+  const content = `
+<d-table sortable>
```

Comment:
Why is the formatting here like this?

Reply:
It seems like that if there are tabs indentation, it will be rendered as code...I haven't figured out why.

Reply:
Just a side note that may not solve the problem. Single backticks are used for template literals, perhaps you could try using triple backticks instead ` ``` ` which are meant for multiline strings in JS

Reply:
> Why is the formatting here like this?

I think this is the same bahaviours for all the Vue components.